gui/slick/views/restart.mako

```diff
@@ -1,3 +1,67 @@
-<%include file="/inc_top.mako"/>
-<%include file="/restart_bare.mako"/>
-<%include file="/inc_bottom.mako"/>
+<%inherit file="/layouts/main.mako"/>
+<%block name="scripts">
+<%
+    try:
+        curSBHost = sbHost
+        curSBHttpPort = sbHttpPort
+        curSBHttpsEnabled = sbHttpsEnabled
+        curSBHandleReverseProxy = sbHandleReverseProxy
+        themeSpinner = sbThemeName
+    except NameMapper.NotFound:
+        curSBHost = "localhost"
+        curSBHttpPort = sickbeard.WEB_PORT
+        curSBHttpsEnabled = "False"
+        curSBHandleReverseProxy = "False"
+        themeSpinner = sickbeard.THEME_NAME
+%>
+<script type="text/javascript" charset="utf-8">
+sbRoot = "${sbRoot}";
+sbHttpPort = "${curSBHttpPort}";
+sbHttpsEnabled = "${curSBHttpsEnabled}";
+sbHandleReverseProxy = "${curSBHandleReverseProxy}";
+sbHost = "${curSBHost}";
+sbDefaultPage = "${sbDefaultPage}";
+</script>
+<script type="text/javascript" src="${sbRoot}/js/lib/jquery-1.11.2.min.js?${sbPID}"></script>
+<script type="text/javascript" src="${sbRoot}/js/restart.js?${sbPID}&amp;${sbDefaultPage}"></script>
+</%block>
+<%block name="content">
+<%
+    try:
+        curSBHost = sbHost
+        curSBHttpPort = sbHttpPort
+        curSBHttpsEnabled = sbHttpsEnabled
+        curSBHandleReverseProxy = sbHandleReverseProxy
+        themeSpinner = sbThemeName
+    except NameMapper.NotFound:
+        curSBHost = "localhost"
+        curSBHttpPort = sickbeard.WEB_PORT
+        curSBHttpsEnabled = "False"
+        curSBHandleReverseProxy = "False"
+        themeSpinner = sickbeard.THEME_NAME
+%>
```

Comment:
It's needed for now as I'm not sure why caching <!% only works on some stuff and not on others. If you can figure that out then we can remove it otherwise neither section seems to be able to access the stuff in there.